packages/fuels/src/cli/commands/build/buildSwayProgram.test.ts

```diff
@@ -58,7 +59,7 @@ describe('buildSwayPrograms', () => {
     vi.spyOn(process.stdout, 'write').mockReturnValue(true);
     configureLogging({ isLoggingEnabled: true, isDebugEnabled: false });
 
-    await buildSwayProgram(fuelsConfig, '/any/workspace/path');
+    await buildSwayProgram(fuelsConfig, '/any/workspace/path', BuildMode.RELEASE);
```

Comment:
Probably over kill, but should one of these tests still be defaulting to `debug`?